core/app/models/spree/stock/package.rb

```diff
@@ -63,7 +63,7 @@ def currency
       end
 
       def shipping_categories
-        Spree::ShippingCategory.joins(products: :variants_including_master).
+        Spree::ShippingCategory.includes(products: :variants_including_master, shipping_methods: [:calculator, zones: :zone_members]).
```

Comment:
Doing includes in a model method is not good practice - it means you'll have no access to this association without loading these other tables. Better to make an includes scope and chain it at the point where the data is used.